lib/capistrano/tasks/maintenance.rake

```diff
@@ -0,0 +1,42 @@
+namespace :forkcms do
+  namespace :maintenance do
+    desc 'Enable maintenance mode'
+    task :enable do
+      on roles(:web) do
+        create_maintenance_folder
+        execute :rm, '-rf', "#{fetch :document_root} && ln -sf #{shared_path}/maintenance #{fetch :document_root}"
+      end
+    end
+
+    desc 'Disable maintenance mode'
+    task :disable do
+      on roles(:web) do
+        execute :rm, '-rf', "#{fetch :document_root} && ln -sf #{current_path} #{fetch :document_root}"
+      end
+    end
+
+    private
+    # Creates the maintenance folder based on the local maintenance folder to display when migrating
+    def create_maintenance_folder
+      # Check if the maintenance folder exists.
+      maintenanceFolderExists = capture("if [ -d #{shared_path}/maintenance ]; then echo 'yes'; fi").chomp
+
+      # Stop if the maintenance folder exists
+      if maintenanceFolderExists == 'yes'
```

Comment:
Eigenlijk is die check dom, want je zou eigenlijk moeten checken of de inhoud van de map klopt.
Nu goed, zou het hetzelfde doen

Reply:
35b8f2e